### DISCUSSION TITLE
Is there a "full-verbose" mode?

### DISCUSSION THREAD

**Discussion by TiagoTiago** [oobabooga/text-generation-webui#6383] [Q&A]
Without skipping the middle on long contexts, and including the output as well.

**Comment by RuneDrawer** [comment]
Really would like to know it as well.

**Comment by TheLounger** [comment]
Don't think there is, but you *could* increase the `max_chars` in [text_generation.py](https://github.com/oobabooga/text-generation-webui/blob/cc8c7ed2093cbc747e7032420eae14b5b3c30311/modules/text_generation.py#L458) by yourself.

**Reply by TiagoTiago** [reply]
Ok, I just put a huge number there as the default value. That takes care of the prompt for most purposes, thanx. (though, the ideal solution would involve checking for the verbose mode, and removing the print-out limit when it's active IMO; I'll see if I can remember to try to figure that out when I have time some other day)

How about the generated text?  Would you happen to know how I can get it streamed to the terminal as it is coming, or at least output once the generation finishes/is interrupted? I have the impression older versions had that behavior.

**Comment by jfmherokiller** [comment]
for the sake of your terminal probably not but you can have it append to a file with 2 lines by editng print_prompt like so.

```
def print_prompt(prompt, max_chars=2000):
    DARK_YELLOW = "\033[38;5;3m"
    RESET = "\033[0m"
    with open("promptStuff.txt", "a",encoding="utf8") as file:
        file.write(prompt)
    if len(prompt) > max_chars:
        half_chars = max_chars // 2
        hidden_len = len(prompt[half_chars:-half_chars])
        hidden_msg = f"{DARK_YELLOW}[...{hidden_len} characters hidden...]{RESET}"
        print(prompt[:half_chars] + hidden_msg + prompt[-half_chars:])
    else:
        print(prompt)

    print()
```

**Reply by jfmherokiller** [reply]
just be warned that you may need to use the chardet or some other encoding detector/set the encoding correctly if you aren't sticking to pure English. (Tho its possible for it to mess you up even if you are using pure English because of the nature of AI)
